setup.py

```diff
@@ -61,7 +61,7 @@
     author_email='jonwayne+oauth2client@google.com',
     url='http://github.com/google/oauth2client/',
     install_requires=install_requires,
-    packages=find_packages(),
+    packages=find_packages(exclude=("tests",)),
```

Comment:
Please use `tests*` to avoid potentially discovering sub-packages of `tests`.

Also, this is a nit, but can you use single quotes here like the rest of the module?